catchup/service.go

```diff
@@ -498,7 +499,14 @@ func (s *Service) pipelinedFetch(seedLookback uint64) {
 			// could resume with the catchup.
 			if s.ledger.IsWritingCatchpointDataFile() {
 				s.log.Info("Catchup is stopping due to catchpoint file being written")
-				s.suspendForCatchpointWriting = true
+				s.suspendForLedgerOps = true
+				return
+			}
+
+			// if the ledger has too many non-flushed account changes, stop catching up to reduce the memory pressure.
+			if !s.ledger.Available() {
```

Comment:
I'm surprised you didn't call `Available` something similar to other case, like `IsBehindCommittingDeltas`. I don't mind too much either way though.

Oh, maybe it's because it covers two issues - lots of deltas, or currently committing accounts?

Reply:
I'm fine renaming, sure.